sermant-agentcore/sermant-agentcore-implement/pom.xml

```diff
@@ -173,6 +174,11 @@
             <artifactId>grpc-protobuf</artifactId>
             <version>${grpc.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-registry-prometheus</artifactId>
+            <version>${micrometer.version}</version>
+        </dependency>
```

Comment:
you should add licence of new dependency under https://github.com/sermant-io/Sermant/tree/develop/LICENSES/vendor

Reply:
fixed